mappings/net/minecraft/item/Item.mapping

```diff
@@ -101,7 +121,11 @@ CLASS net/minecraft/class_1792 net/minecraft/item/Item
 	METHOD method_7875 byRawId (I)Lnet/minecraft/class_1792;
 		ARG 0 id
 	METHOD method_7876 getTranslationKey ()Ljava/lang/String;
+		COMMENT Gets the translation key of this item.
 	METHOD method_7877 isIn (Lnet/minecraft/class_1761;)Z
+		COMMENT Checks if this item should appear in a specified item group.
+		COMMENT
+		COMMENT @return true if the item is in the specified item group or the item group is {@link net.minecraft.item.ItemGroup#SEARCH}.
```

Comment:
Is there a special case for the `SEARCH` item group? I think this method just checks whether the item appears at all within the item group.

Reply:
Yes, the search item group is a special case

Reply:
This is really `shouldBeAddedTo` or so as this is called when items are checked to be whether to added to groups, not when groups are ready and to query if an item is in a group